CHANGES.md

```diff
@@ -78,7 +78,9 @@ OpenSSL 3.0
    *Tomáš Mráz*
 
  * Removed RSA padding mode for SSLv23 (which was only used for
-   SSLv2)
+   SSLv2). This includes the functions RSA_padding_check_SSLv23() and
+   -RSA_padding_add_SSLv23() and the `-ssl` option in the deprecated
```

Comment:
Oops there is also a spurious `-` before RSA_padding_add_SSLv23.

Reply:
updated fixup commit pushed.

(four word-sentence, 3 of them jargon :)